plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -211,6 +264,54 @@ fu_cros_ec_usb_device_flush (FuDevice *device, gpointer user_data,
 	return TRUE;
 }
 
+/*
+ * Channel TPM extension/vendor command over USB. The payload of the USB frame
+ * in this case consists of the 2 byte subcommand code concatenated with the
+ * command body. The caller needs to indicate if a response is expected, and
+ * if it is - of what maximum size.
+ */
+static gboolean
+fu_cros_ec_usb_ext_cmd (FuDevice *device, guint16 subcommand,
+			gpointer cmd_body, gsize body_size,
+			gpointer resp, gsize *resp_size,
+			gboolean allow_less, GError **error)
+{
+	FuCrosEcUsbDevice *self = FU_CROS_EC_USB_DEVICE (device);
+	guint16 *frame_ptr;
+	gsize usb_msg_size;
+	g_autofree struct update_frame_header *ufh;
+
+	usb_msg_size = sizeof (struct update_frame_header) +
+		       sizeof (subcommand) + body_size;
+
+	ufh = g_malloc0 (usb_msg_size);
+
+	ufh->block_size = GUINT32_TO_BE (usb_msg_size);
+	ufh->cmd.block_digest = 0;
+	ufh->cmd.block_base = GUINT32_TO_BE (UPDATE_EXTRA_CMD);
+	frame_ptr = (guint16 *)(ufh + 1);
+	*frame_ptr = GUINT16_TO_BE (subcommand);
+
+	if (body_size != 0) {
+		gsize offset =  sizeof (struct update_frame_header) + sizeof (subcommand);
+		if (!fu_memcpy_safe ((guint8 *) ufh, usb_msg_size, offset,
+				     (const guint8 *) cmd_body, body_size,
+				     0x0, body_size, error))
+			return FALSE;
+	}
+
+	if (!fu_cros_ec_usb_device_do_xfer (self, (guint8 *)ufh, usb_msg_size,
+					    (guint8 *)resp,
+					    resp_size != NULL ? *resp_size : 0,
+					    TRUE, NULL, error)) {
+
```

Comment:
extra whitespace here

Reply:
if you're not going to output anything or prefix I would just `return fu_cros_ec_usb_device_do_xfer `

Reply:
Done.